### PR TITLE
Quotes

### PR DESCRIPTION
**Pull request by kr3cj**
Happy to remove any disparaging ones

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,10 +39,31 @@
...
 * Ubuntu
 * Complainy Hulu customers
 * Places called "Fish Tacos"
 * Going south of Alameda
+
+## Quotes
+* Wait a tic
+* Al, I'll miss you the most
+* I have the sys admin physique
+* Words have meaning!
```
Comment (elementalvoid) [inline README.md]: This was me, not @rpkish.

File: README.md [hunk 2]
```diff
@@ -39,10 +39,30 @@
...
 * Places called "Fish Tacos"
 * Going south of Alameda
+
+## Quotes
+* Wait a tic
+* Al, I'll miss you the most
+* I have the sys admin physique
+* Everything's coming up Milhouse
+* It's not "slash user", it's "slash U, S, R"
+* Cold food is for peasants
```
Comment (rpkish) [inline README.md]: Cold sandwiches are for peasants. Words do indeed have meaning.

File: README.md [hunk 3]
```diff
@@ -39,10 +39,30 @@
...
+* Subarus are so hot
+* Davita drivers are asshats!
+* Get off my lawn!
+* Let's go back. First, there were the dinosaurs
+* I hate java
+* Douche nozzle
+* Dude, stop cutting into our ESOP
+* Go f\*ck yourself!
+* F\*ck you, f\*ck you, you're cool, and f\*ck you
```
Comment (rpkish) [inline README.md]: this is technically a re-quote, but I guess it can slide.

**Review by rpkish — CHANGES_REQUESTED** [review]
s/food/sandwiches/ and I will convert this to a +1

File: README.md (hunk 2 above)
Comment (elementalvoid) [reply README.md]: I distinctly remember:
>Cold food is for peasants. We invented fire for a reason.

While you may agree about words; there's a reason y'all called me "Words-have-meaning-Matt". :expressionless: Just sayin'.

Comment (rpkish) [reply README.md]: For what it's worth, human memory is fallible, even yours my good Sir. This statement misrepresents my actual feelings on cold food. I have made the standard line on fire countless times, in regards to sandwiches. Words do indeed have meaning, food includes many items which have no business being heated, at least in their final form:
 - Beer 
 - Ice cream
 - Chocolate milk

Comment (kr3cj) [reply README.md]: `Cold sandwiches are for peasants` doesn't quite have the same ring to it so I think I'll just remove the line. FWIW, my memory is in alignment with @elementalvoid's on this one.

File: README.md (hunk 3 above)
Comment (kr3cj) [reply README.md]: True. I'd like to keep this original. WIll remove.

**Review by rpkish — APPROVED** [review]
Changes look good. Approved!

File: README.md (hunk 2 above)
Comment (elementalvoid) [reply README.md]: I feel the need to sit in a triangle and vehemently agree ; )

I'll split the difference: while I recall the quote differently than @rpkish I _do agree_ it was always made _in reference to sandwiches_ (as that was always the conversation topic).

**Review by rpkish — APPROVED** [review]

**Pull request merged**